bin/plotting/pycbc_plot_bank_corner

```diff
@@ -78,10 +78,6 @@ parser.add_argument("--parameters",
          "property of that parameter will be used. If not "
          "provided, will plot all of the parameters in the "
          "bank.")
-parser.add_argument("--log-parameters",
-    nargs="+",
-    default=[],
-    help="Plot these parameters on a log scale")
```

Comment:
Is there a reason the `log-parameters` argument was removed? I don't see how this is related to the rest of the changes in this MR

Reply:
I had missed this. We definitely want to keep this option, since a hyperbank-like bank cannot be meaningfully displayed in linear component masses or total mass.

Reply:
This is a somewhat-related bug, as the supervision script calls this script. The option isn't actually used, if wanting a log parameter, we would have to pass "log10(mchirp)" (or similar) to `--parameters`